### PR TITLE
Translations: add key screenshots

### PR DESCRIPTION
**Pull request by mrtnzlml**
Note: automatic deployment into PhraseApp is not ready yet.

OMG, this was painful... 🙂

### CODE REVIEW

File: app/translations/src/DefaultVocabulary.js [hunk 1]
```diff
@@ -26,74 +26,71 @@
  * TODO: better parameters structure (native code is not replacing it anyway)
  */
 const Translations = {
```
Comment (mrtnzlml) [inline app/translations/src/DefaultVocabulary.js]: I just sorted the lines by alphabet.

**Comment by mrtnzlml** [comment]
@tbergq Please wait - I need to verify the LFS.

**Comment by tbergq** [comment]
Ok, ping me again when it is ready

**Comment by mrtnzlml** [comment]
@tbergq Please check it. Now it's actually on LFS.

**Pull request merged**